src/internal.rs

```diff
@@ -114,8 +114,8 @@ impl Global {
         for local in self.locals.iter(&guard) {
             match local {
                 Err(IterError::Stalled) => {
-                    // The iteration is stalled by another thread's iteration. Since that thread
-                    // also tries to advance the epoch, we leave the job to that thread.
+                    // A concurrent thread stalled this iteration. Since it also tries to advance
```

Comment:
The concurrent thread may not try to advance the global epoch; it may just delete a node.

Reply:
Since we can't tell whether the concurrent thread is just deleting a node or actually advancing the epoch, should the code stay the same and the comment just be updated to document this?